src/hotspot/os/windows/globals_windows.hpp

```diff
@@ -36,6 +36,9 @@
                          range,                                           \
                          constraint)                                      \
                                                                           \
+product(bool, UseAllProcessorGroups, false,                               \
```

Comment:
Maybe we should add "Windows" on the name, as this is only applicable to Windows.

Reply:
I'm not sure but I think if this is a product flag and is set to 'false' by default then you'll need a CSR approval/review.
https://wiki.openjdk.org/display/csr/CSR+FAQs

Reply:
I have renamed the product flag to UseAllWindowsProcessorGroups